src/vm/arm64/CallDescrWorkerARM64.asm

```diff
@@ -76,6 +77,8 @@ LNoFloatingPoint
         ldr     x9, [x19,#CallDescrData__pTarget]
         blr     x9
 
+        ldr     x19, [fp, 24] ; Fixup corrupted X19 callee preserved register
```

Comment:
x19 is callee saved so it should not be trashed on return from call. Can you explain who/when is x19 getting corrupted?